README.md

```diff
@@ -18,6 +18,9 @@ We know it can be difficult to get into the whole open source thing at first. To
 Random halloween facts are posted regularly.  
 !hallofact - Show the last posted Halloween fact
 
+### Avatar Spookification
+!savatar - Spookifies your avatar with a few random effects.
```

Comment:
Hacktoberbot's command prefix is `.`, might as well fix the entire README in this PR